tests/sentry/ingest/billing_metrics_consumer/test_billing_metrics_consumer_kafka.py

```diff
@@ -5,101 +5,133 @@
 
 from arroyo.backends.kafka import KafkaPayload
 from arroyo.types import BrokerValue, Message, Partition, Topic
+from sentry_kafka_schemas.schema_types.snuba_generic_metrics_v1 import GenericMetric
 
 from sentry.constants import DataCategory
-from sentry.ingest.billing_metrics_consumer import (
-    BillingTxCountMetricConsumerStrategy,
-    MetricsBucket,
-)
+from sentry.ingest.billing_metrics_consumer import BillingTxCountMetricConsumerStrategy
+from sentry.models.project import Project
+from sentry.sentry_metrics import indexer
 from sentry.sentry_metrics.indexer.strings import SHARED_TAG_STRINGS, TRANSACTION_METRICS_NAMES
-from sentry.testutils.helpers.datetime import freeze_time
+from sentry.sentry_metrics.use_case_id_registry import UseCaseID
+from sentry.testutils.pytest.fixtures import django_db_all
 from sentry.utils import json
 from sentry.utils.outcomes import Outcome
 
 
+@django_db_all
 @mock.patch("sentry.ingest.billing_metrics_consumer.track_outcome")
-@freeze_time("1985-10-26 21:00:00")
```

Comment:
i believe this was added intentionally by possibly devinfra/asottile, and you should get in touch with them to ensure that this is the way to go.